onnxruntime/python/onnxruntime_validation.py

```diff
@@ -8,6 +8,7 @@
 import platform
 import linecache
 import warnings
+import sys
 
 
 def check_distro_info():
```

Comment:
>check_distro_info [](http://example.com/codeflow?start=4&length=17)

Seems could use util

Reply:
it is a different platform - this one is from python